include/audio/aseq.h

```diff
@@ -220,21 +231,190 @@
 #define FONTANY_INSTR_ASM_NOISE    136
 
 
-
-#ifdef _LANGUAGE_ASEQ
-
 /**
- *  MML Version
- */
-
-#ifndef MML_VERSION
-    #error "MML version not defined, define MML_VERSION in the cpp invocation"
+ *  Command Opcode IDs
+ */
+
+// control flow commands
+#define ASEQ_OPC_CONTROL_FLOW_FIRST 0xF2
+#define ASEQ_OPC_CTRLFLOW_RBLTZ     0xF2
+#define ASEQ_OPC_CTRLFLOW_RBEQZ     0xF3
+#define ASEQ_OPC_CTRLFLOW_RJUMP     0xF4
+#define ASEQ_OPC_CTRLFLOW_BGEZ      0xF5
+#define ASEQ_OPC_CTRLFLOW_BREAK     0xF6
+#define ASEQ_OPC_CTRLFLOW_LOOPEND   0xF7
+#define ASEQ_OPC_CTRLFLOW_LOOP      0xF8
+#define ASEQ_OPC_CTRLFLOW_BLTZ      0xF9
+#define ASEQ_OPC_CTRLFLOW_BEQZ      0xFA
+#define ASEQ_OPC_CTRLFLOW_JUMP      0xFB
+#define ASEQ_OPC_CTRLFLOW_CALL      0xFC
+#define ASEQ_OPC_CTRLFLOW_DELAY     0xFD
+#define ASEQ_OPC_CTRLFLOW_DELAY1    0xFE
+#define ASEQ_OPC_CTRLFLOW_END       0xFF
+
+// sequence commands
+#define ASEQ_OPC_SEQUENCE_TESTCHAN          0x00 // low nibble used as argument
```

Comment:
Some of these get pretty long, imo `OPC` -> `OP` is sufficient and more controversially I think writing `SEQUENCE`/`CHANNEL` in full is not really necessary either, common abbreviations are `SEQ`/`CHAN`. The control flow instructions that are common to all code sections probably don't need a qualifier at all, e.g. `ASEQ_OP_RBLTZ`.